components/org.wso2.carbon.identity.api.server.input.validation/org.wso2.carbon.identity.api.server.input.validation.v1/src/main/java/org/wso2/carbon/identity/api/server/input/validation/v1/core/ValidationRulesManagementApiService.java

```diff
@@ -107,6 +126,33 @@ public List<ValidationConfigModel> updateInputValidationConfiguration(
         }
     }
 
+    /**
+     * Method to update input validation configuration.
+     *
+     * @param validationConfigModel Validation Configuration Model.
+     * @param tenantDomain          Tenant domain name.
+     * @return ValidationConfigModel for the field.
+     */
+    public ValidationConfigModel updateInputValidationConfigurationForField(
+            ValidationConfigModel validationConfigModel, String tenantDomain) {
+
+        try {
+            isFieldFound(validationConfigModel.getField());
+            List<ValidationConfigModel> configModels = new ArrayList<>();
+            configModels.add(validationConfigModel);
+            List<ValidationConfiguration> requestDTO = buildRequestDTOFromValidationRequest(configModels);
+            validateProperties(requestDTO, tenantDomain);
+            ValidationConfiguration configurations = InputValidationServiceHolder.getInputValidationMgtService()
+                    .updateValidationConfiguration(requestDTO.get(0), tenantDomain);
```

Comment:
Validate whether the size of the requestDTO can be 0. If so handle it

Reply:
RequestDTO can be empty list. It is validate it in `validateProperties(requestDTO, tenantDomain)` (previous line)